cnd/src/db.rs

```diff
@@ -132,6 +133,16 @@ pub enum Error {
     SwapNotFound,
 }
 
+// Initial attempt at a created swap, this is only for a single protocol pair.
+#[derive(Debug, Clone)]
+pub struct CreatedSwap<A, B> {
+    pub swap_id: LocalSwapId,
+    pub alpha: A,
+    pub beta: B,
+    pub peer: DialInformation,
```

Comment:
I think we only care about the peer id, the address hint is unlikely to be valid in the long run.

Reply:
I guess the question is what recovery behaviour if the node crash between swap creation and before finalizing.
If we have no hope of recovering, the address hint is indeed useless.
If a node that starts with a swap created but not finalised would try to automatically recover by re-initiating the announcement communication then the address hint will be needed.

Reply:
Good question. I would still separate the information and maybe have an on-disk cache of peer-id <-> addresses somewhere that this would go in!

Reply:
Sounds like a feature to me though so I would not save the information for now to keep the scope small.